mewarpx/mewarpx/mwxrun.py

```diff
@@ -68,20 +64,54 @@ def init_grid(self, lower_bound, upper_bound, number_of_cells, **kwargs):
                 of list has to equal number of dimensions.
             upper_bound (list): Maximum coordinates for all direction; length
                 of list has to equal number of dimensions.
-            cell_nums (list): Number of grid cells in each direction.
+            number_of_cells (list): Number of grid cells in each direction.
             kwargs (dict): Dictionary containing extra arguments. These can
                 include:
+                - ``use_rz`` to set the simulation to use cylindrical
+                  coordinates.
```

Comment:
My vote is to just put `use_rz=False` as an explicit function argument, not part of the `kwargs` dict. Functionality will be the same and it will be more evident to people.

Reply:
What if we made geometry string an explicit argument, with the docstring specifying the allowed strings. It would help simplify the logic later, and it would communicate better to the user the exact allowed geometries.